src/boostvqe/boost.py

```diff
@@ -38,41 +37,135 @@
     train_vqe,
 )
 
+NSHOTS = 100
 
-def main(args):
-    """VQE training."""
-    # set backend and number of classical threads
 
-    if args.platform is not None:
-        qibo.set_backend(backend=args.backend, platform=args.platform)
-    else:
-        qibo.set_backend(backend=args.backend)
-        args.platform = GlobalBackend().platform
+def main(
```

Comment:
At this point I will not call it `main` anymore.
Perhaps `boostvqe`, `boost` or even `dbqaxvqe`?